CHANGELOG.md

```diff
@@ -18,6 +18,9 @@ Please avoid adding duplicate information across this changelog and JIRA/doc inp
 ### Deprecated Features
 - ROX-12620: We continue to simplify access control management by grouping some permissions in permission sets. As a result:
   - The permission `WorkflowAdministration` will deprecate the permissions `Policy, VulnerabilityReports`.
+- ROX-14398: We continue to simplify access control management by grouping some permissions in permission sets. As a result:
+  - The permission `Access` will deprecate the permissions `Role`.
+  - The default role `Scope Manager` will be removed.
```

Comment:
We've added the `required actions` section below to address some feedback regarding customers not necessarily knowing what they have to do to remediate the deprecation. We should cover the role deprecation within there as well.

Especially with the removal of the `Scope Manager` role, this _does_ require an action if its used as a default minimum access role / group within auth providers.